src/component/SearchArea/SearchArea.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useMediaQuery } from 'react-responsive';
+import SearchAreaStyled, { Title, SearchBar } from './SearchArea.styles.jsx';
+import SearchIcon from '../../assets/icon_search.svg';
+import RecommendArea from '../RecommendArea/RecommendArea.jsx';
+
+const SearchArea = () => {
+  const isPC = useMediaQuery({ query: '(min-width: 1040px)' });
```

Comment:
`Question`
궁금해서 여쭈어 보는 내용입니다. 
PC와 모바일을 구분하는 가장 좋은 방법이 어떤 거라고 생각하시나요? 
저는 절대적인 폭을 계산하는것도 하나의 방법이지만,
가로의 세로의 비율을 확인하는 방법이 좀더 효과적인 방법이라고 생각 합니다.
보통 저는 이렇게 적용하는데요. 혹시 이렇게 지정하신 이유를 알 수 있을까요?
```suggestion
  const isPC = useMediaQuery({ query: '(max-aspect-ratio: 3/4)' });
```

Reply:
좋은 질문 감사합니다!

https://clinicaltrialskorea.com/ 
위 사이트에서 폭 1040px 기준으로 반응형 분기가 이루어지고 있는 점 참고해서 적용했습니다 
정확히 pc와 모바일에 따른 분기가 아닌데 제가 변수명을 좀 부적절하게 지은 것 같네요

그리고 가로와 세로 비율을 분기점으로 삼는 방법은 사용해본 적이 없는데, 
모바일/pc 분기 외에
데스크탑 화면에서 전체화면이 아닌 경우나 태블릿 화면에 대해서도 혹시 적용이 가능한건가요?